### ISSUE TITLE
[SDL3] Be able to retrieve real rendering backend name when using GPU backend

### ISSUE THREAD

**Issue by MrOnlineCoder** [libsdl-org/SDL#11600]
If renderer is created with `SDL_CreateRenderer(window, NULL)` and SDL decides to use the new GPU API for it, the call to `SDL_GetRendererName` function will return `gpu` as the result for it, giving not much usefulness in terms of logging or debugging about which backend/graphics API is used behind the scenes. Maybe it would be worth adding it as property to renderer?

**Comment by AntTheAlchemist** [comment]
Can we just add the backend renderer in brackets to 'SDL_GetRendererName'? So we'd get "GPU (OpenGL)".

**Comment by MrOnlineCoder** [comment]
That's one of solutions, at least I personally use it for mostly debugging and including in crash reports / logs, so that would be just enough. And I am not sure that renderer name is something that should be parsed programatically, so making it more human-readable like you suggested is fine, I guess?

**Comment by icculus** [comment]
No, let's make it a property. The name strings probably shouldn't be messed with like that.

**Comment by slouken** [comment]
Done!

Check https://github.com/libsdl-org/SDL/commit/969f03d9f0b20a32b1e03cc181018c5dc3d5c317 for an example of getting the GPU driver from the renderer.
